### PR TITLE
Add whatsnew_69.lang

### PR DESCRIPTION
**Pull request by craigcook**
Work-in-progress page at https://www-demo1.allizom.org/firefox/69.0/whatsnew/

### CODE REVIEW

File: en-US/firefox/whatsnew_69.lang [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+;Your Firefox Browser now automatically blocks online snoopers
+Your Firefox Browser now automatically blocks online snoopers
+
+
+;Now it’s harder than ever for spying companies to track you online. That’s because Enhanced Tracking Protection — a feature that blocks known tracking cookies — is on by default in your updated browser.
+Now it’s harder than ever for spying companies to track you online. That’s because Enhanced Tracking Protection — a feature that blocks known tracking cookies — is on by default in your updated browser.
+
+
```
Comment (peiying2) [inline en-US/firefox/whatsnew_69.lang]: Add a comment that Pocket is a product name and should remain in English. Pocket is not offered in most of the markets that the mozorg site is localized in.

**Comment by craigcook** [comment]
Good call. Added comments.

**Comment by craigcook** [comment]
Added to sources in https://github.com/mozilla-l10n/langchecker/pull/904

This page is only being shown to English, French, and German. We're also working on a new default whatsnew page for all locales that won't change as frequently, and it doesn't mention Pocket or Monitor. I hope to have those for you strings in a few days.

**Comment by peiying2** [comment]
I have approved the langchecker PR. Please go ahead and add locale to files, merge, and prod push.

**Comment by peiying2** [comment]
Updated langchecker PR approved. Pls proceed.

**Pull request merged**